### PR TITLE
CMake: propagate CUDA enablement

### PR DESCRIPTION
**Pull request by keithc-ca**
* set CUDA_BIN_PATH so find_package(CUDA) uses the right version
* set OMR_OPT_CUDA=ON

### CODE REVIEW

**Comment by keithc-ca** [comment]
Without these changes, OMR tries to use `/usr/local/cuda` which may not exist or may be a version other than what the user specified via the `--with-cuda=` configure option.

FYI @dnakamura

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -493,8 +493,15 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 
   CMAKE_ARGS += $(EXTRA_CMAKE_ARGS)
 
+  ifeq (true,$(OPENJ9_ENABLE_CUDA))
+  	CMAKE_ARGS += -DOMR_OPT_CUDA=ON
+    CMAKE_CUDA_ENV := CUDA_BIN_PATH="$(CUDA_HOME)"
```
Comment (dnakamura) [inline closed/OpenJ9.gmk]: Looks like mixed tabs & spaces here

Comment (keithc-ca) [reply closed/OpenJ9.gmk]: Yes - I'll fix that.

Comment (keithc-ca) [reply closed/OpenJ9.gmk]: Done.

**Comment by pshipton** [comment]
jenkins compile xlinuxcm jdk11

**Pull request merged**